### ISSUE TITLE
Progress Logging does not appear in cluster manager output

### ISSUE THREAD

**Issue by astro-kevin** [TuringLang/Turing.jl#1916]
I'm sampling a Turing model in a cluster managed by SLURM. The progress bar generated from log-progress doesn't seem to appear until the sampling is all finished. I know it isn't to do with the progressbars generally because I use `ProgressMeters` elsewhere in my code. But I'm not sure whether this is to do with Turing or with `ProgressLogging.jl`.

**Comment by devmotion** [comment]
Maybe you are facing https://github.com/TuringLang/AbstractMCMC.jl/issues/82?

**Comment by astro-kevin** [comment]
I don't believe I am. I'm not sampling multiple chains at once, but only a single chain.

**Comment by devmotion** [comment]
I see. In that case everything should work automatically if one does not explicitly use a logger with a log level less than or equal to `ProgressLogging.ProgressLevel` (in that case users have to make sure that their logger handles progress logs correctly). On Windows with Julia <= 1.5.3 and with IJulia the default progress logging backend used by AbstractMCMC is ConsoleProgressMonitor, and otherwise TerminalLoggers is used.

You could try to check if the minimal progress logging examples, e.g., in the docs of TerminalLoggers, work correctly on the cluster (https://julialogging.github.io/TerminalLoggers.jl/dev/#Progress-bars-1).

**Comment by damonbayer** [comment]
When you say they don't appear are you talking about an interactive session or a scheduled job with an output file?
